server/models/contests/getAllQuestions.js

```diff
@@ -30,11 +30,6 @@ function getAllQuestions({ username, params }) {
         if (error || results === undefined) {
           return reject(error)
         }
-        if (!results || !results.length) {
-          return reject(
-            'You do not have moderator access of the contest. If the contest is active, make sure that you are registered as a participant.'
-          )
-        }
```

Comment:
why?

Reply:
rest seems fine

Reply:
If there are no questions added in a contest, then also it shows this error. In frontend, I would show the questions page only when the user is registered for that contest, this error message is not needed.